apps/desktop/src/main/lib/agent-setup/templates/copilot-hook.template.ps1

```
@@ -0,0 +1,55 @@
+}
+$Query = ($Fields.GetEnumerator() | ForEach-Object {
+	$value = if ($null -eq $_.Value) { '' } else { [string]$_.Value }
+	"{0}={1}" -f [Uri]::EscapeDataString($_.Key), [Uri]::EscapeDataString($value)
+}) -join '&'
+
+try {
+	Invoke-WebRequest -Uri "http://127.0.0.1:$Port/hook/complete?$Query" `
+		-Method Get -UseBasicParsing -TimeoutSec 2 -ErrorAction Stop | Out-Null
+} catch {
+	# Silent — the agent must not be blocked by transient notification failures.
+}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)